Configuration.md

```diff
@@ -198,16 +198,15 @@ target_template=gs://ecmwf-downloads/hres-single-level/{}.nc
 partition_keys=
     date
 [parameters.deepmind]
-api_key=KKKKK1
-api_url=UUUUU1
+secret_key=projects/PROJECT_NAME/secrets/SECRET_NAME/versions/1
```

Comment:
Please add a separate section that demos secrets mgr. Users should see examples of both ways of adding keys.